presto-parser/src/main/java/io/prestosql/sql/tree/AddColumn.java

```diff
@@ -27,22 +27,26 @@
 {
     private final QualifiedName name;
     private final ColumnDefinition column;
+    private final boolean tableNameExists;
```

Comment:
tableExists here and in other statement classes

Reply:
fixed thanks!